sonar-kotlin-plugin/src/main/java/org/sonarsource/kotlin/plugin/KotlinSensor.kt

```diff
@@ -58,12 +59,23 @@ class KotlinSensor(
 ): AbstractKotlinSensor(
     checkFactory, language, KOTLIN_CHECKS
 ) {
+    val telemetryData = TelemetryData()
```

Comment:
We assume the lifetime of the sensor spans the entire analysis, or even longer than that (in the case of SonarQube IDE, where the sensor lifespan may outlive the single analysis).

If the assumption is broken, a `false` (`0`) value for `kotlin.android` may override a `true` (`1`) value, previously sent for the same project and in the same analysis, by another instance of the sensor.